File: Magazyn/Magazyn/Tables/Adres.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Magazyn.Tables
{
    public class Adres
    {
        public int idAd { set; get; }
        public string street { set; get; }
        public string number { set; get; }
        public string city { set; get; }
        public string pcode { set; get; }
    }
}

File: apiForMagazyn/src/main/java/org/apiForMagazyn/api/adres/Adres.java
package org.apiForMagazyn.api.adres;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import javax.validation.constraints.Size;

@Entity
@Table
@Setter
@Getter
public class Adres {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idAd;

    @Size(max = 50)
    private String street;

    @Size(max = 8)
    private String number;

    @Size(max = 50)
    private String city;

    @Size(max = 10)
    private String pCode;

}
